lesson_09/oop/oop_app/src/test/java/com/codedifferently/lesson9/sherllin/BookTest.java

```diff
@@ -0,0 +1,123 @@
+package com.codedifferently.lesson9.sherllin;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+// We want to define a book and keep track of its title, isbn, author(s), number of pages, and
+// whether it is checked out or not.
+// We want define a patron with properties including their name and the books they have checked out.
+// We want to define a library that:
+// Allows us to add/remove a book to and from its collection of books.
+// Allows us to register new patrons
+// Allows us to check out books to patrons.
+// Allows us to return books from patrons.
+
+public class BookTest {
+
+  @Test
+  public void testBookProperties() {
+    // Arrange
+    String title = "The Alchemist";
+    long isbn = 9780061122415L;
+    String author = "Paulo Coelho";
+    int pages = 208;
+    boolean checkedOut = false;
+
+    // Act
+    Book theAlchemist = new Book(title, isbn, author, pages, checkedOut);
+
+    // Assert
+    assertEquals(title, theAlchemist.getBookTitle());
+    assertEquals(isbn, theAlchemist.getIsbn());
+    assertEquals(author, theAlchemist.getAuthor());
+    assertEquals(pages, theAlchemist.getPages());
+    assertEquals(checkedOut, theAlchemist.checkedOut());
+
+    // Test additional books
+    Book theCatInTheHat = new Book(title, isbn, author, pages, checkedOut);
```

Comment:
Everything after this is redundant since it's testing exactly what you've already asserted above.